src/NuGet.Clients/NuGetConsole.Host.PowerShell/PowerShellHost.cs

```diff
@@ -528,6 +553,12 @@ public bool Execute(IConsole console, string command, params object[] inputs)
                 throw new ArgumentNullException(nameof(command));
             }
 
+            // Increase command execution counters for PMC/PMUI
+            _vsPowerShellHostTelemetryEmit.Value.IncreaseCommandCounter(console is IWpfConsole);
+
+            // Check and add to telemetery if a command is NugetCommand like 'install-package' etc.
+            _vsPowerShellHostTelemetryEmit.Value.IsNuGetCommand(command);
```

Comment:
Can we have separate counters for NuGet PMC commands and non-NuGet PMC commands?

Reply:
During the `spec` discussion decided it's not necessary and just keep indicator. Please not this NuGetCommand only covers basic 10 commands not all PMC commands like ones from EF. Only way to differentiate actual NuGet command from powershell command is manually enumerating all of them. I feel but kind of pointless, since too many of them.